pansat/formats/hdf4.py

```diff
@@ -26,13 +26,13 @@
 
 LOGGER = logging.getLogger(__name__)
 
-try:
-    from pyhdf.HDF import HDF
-    from pyhdf.SD import SD
-    from pyhdf.VS import VS
-except ImportError as error:
-    LOGGER.error("The pyhdf package is required to read HDF4 file. Please install it.")
-    raise error
+# try:
```

Comment:
Is this intended?

Reply:
Ui, good catch! I was trying out to get the documentation somehow into RTD, this did not help. But I will uncomment it of course